services/director/tests/fixtures/fake_services.py

```diff
@@ -157,7 +157,9 @@ async def _build_push_image(
     await _create_base_image(docker_labels, image_tag)
 
     # push image to registry
+    docker = Docker()
     await docker.images.push(image_tag)
+    await docker.close()
```

Comment:
why not just use async with?

Reply:
the director currently uses a library named [apiset](https://github.com/aamalev/aiohttp_apiset). the context manager in aiodocker is not yet available in the compatible version of the library.
Since we will anyway replace the director with a FastAPI version (@pcrespov will have a look) it stays like that for now.